WalletWasabi/Wallets/Wallet.cs

```diff
@@ -73,6 +73,8 @@ public class Wallet : BackgroundService, IWallet
 
 	public event EventHandler<WalletState>? StateChanged;
 
+	private bool NewFiltersEventSubscribed { get; set; }
```

Comment:
Consider removing this. Is it safe to remove delegate if it is not in the invocation list? 

https://learn.microsoft.com/en-us/dotnet/api/system.delegate.remove?view=net-8.0&redirectedfrom=MSDN#System_Delegate_Remove_System_Delegate_System_Delegate_